azafea/event_processors/endless/activation/v1/cli.py

```diff
@@ -100,3 +110,26 @@ def do_parse_images(config: Config, args: argparse.Namespace) -> None:
     progress(num_records, num_records, end='\n')
 
     log.info('All done!')
+
+
+def do_transform_countries_alpha_3_to_2(config: Config, args: argparse.Namespace) -> None:
+    db = Db(config.postgresql)
+    log.info('Transform alpha-3 country codes into alpha-2 for activations')
+
+    with db as dbsession:
+        query = dbsession.query(Activation).filter(func.length(Activation.country) == 3)
+        num_records = query.count()
+
+        if num_records == 0:
+            log.info('-> No activation with alpha-3 country code found')
+            return None
+
+        requests = transform_alpha_3_into_2(Activation.__tablename__)
+        for i, request in enumerate(requests, start=1):
+            dbsession.execute(request)
```

Comment:
Why use raw SQL here, instead of (something like, my sqlalchemy is rusty):

```python3
for alpha2, alpha3 in COUNTRIES:
    dbsession.execute(dbsession.query(Activation).update().where(Activation.country == alpha3).values(country=alpha2)
```

?

Reply:
You’re right…

Reply:
Again, a one-time migration so probably fine as-is, we can see by inspection that the queries as written are OK.

Reply:
> I’ll check the prod data before resolving this conversation.

All the country codes from the ping and activation tables are in the list.